resources/js/Pages/Auth/Login.vue

```diff
@@ -5,6 +5,10 @@ import InputError from '@/Components/InputError.vue';
 import InputLabel from '@/Components/InputLabel.vue';
 import PrimaryButton from '@/Components/PrimaryButton.vue';
 import TextInput from '@/Components/TextInput.vue';
+import SecondaryButton from '@/Components/SecondaryButton.vue';
+import Bitbucket from "@/Components/Icons/Bitbucket.vue";
+import Github from "@/Components/Icons/Github.vue";
+import Google from "@/Components/Icons/Google.vue";
```

Comment:
I like how you've put the icons in their own components, it makes the html a lot cleaner than having svg code filling it up